content/e/mzw/2015-01-11-mzw-charity-show-2015.md

```diff
@@ -20,19 +20,32 @@ In another corner of the country, DDW also held a [charity event](@/e/ddw/2015-0
 ## Card
 
 {% card() %}
-- - '[Szymon Siwiec](@/w/szymon-siwiec.md)'
+- - '[Szymon "Siwy" Siwiec](@/w/szymon-siwiec.md)'
   - '[Kowboj Luke](@/w/red-thunder.md)'
 - - '[Hexia](@/w/hexia.md)'
-  - '[Mario Rodriguez](@/w/kripto.md)'
+  - '[Gracjan "Mario Rodriguez"](@/w/kripto.md)'
+- - '[Shadow](@/w/shadow.md)'
+  - '[Justin Joy](@/w/justin-joy.md)'
+  - g: true
 - - '[Polish Hammer](@/w/jedrus-bulecka.md)'
   - '[Asmund](@/w/asmund.md)'
 - - '[Shadow](@/w/shadow.md)'
   - '[Justin Joy](@/w/justin-joy.md)'
   - r: 'DQ'
 - credits:
     Referee: '[Sędzia Piotr](@/w/mr-b.md)'
+    Ring announcer: '???'
 {% end %}
 
+### Highlights
+
+* After brawling outside the ring for a while, [Szymon Siwiec](@/w/szymon-siwiec.md) threw [Kowboj Luke](@/w/red-thunder.md) back in and sluggishly followed him shortly thereafter. The two then just lied on the mat, earning a laugh and an applause from the audience (as well as a chuckle from the camera operator).
```

Comment:
Laid, I think. https://www.britannica.com/story/lay-lie-lied-lain-when-do-we-use-which